.github/workflows/moving-cards.yml

```diff
@@ -57,20 +57,20 @@ jobs:
       - name: Triage labeling
         # Only if moved into triage
         if: ${{ github.event.project_card.column_id == env.TRIAGE_COLUMN_ID }}
-        uses: andymckay/labeler@1.0.4
+        uses: fmulero/labeler@1.0.5
         with:
           add-labels: triage
           remove-labels: on-hold, in-progress, solved
       - name: From Bitnami labeling
         if: ${{ github.event.project_card.column_id == env.BITNAMI_COLUMN_ID }}
-        uses: andymckay/labeler@1.0.4
+        uses: fmulero/labeler@1.0.5
         with:
           add-labels: ${{ (needs.get-issue.outputs.author == 'bitnami-bot' && needs.get-issue.outputs.type == 'pull_request') && 'automated, auto-merge' || 'bitnami' }}
           remove-labels: on-hold, in-progress, triage, solved
       - name: Verify labeling
         # Only if moved into bitnami column and the PR is ready for review
         # This step uses a github-script to add the label intentionally.
-        # Consecutive calls to andymckay/labeler@1.0.4 can remove previous assigned labels, see https://github.com/andymckay/labeler/issues/40
+        # Consecutive calls to fmulero/labeler@1.0.5 can remove previous assigned labels, see https://github.com/fmulero/labeler/pull/2
```

Comment:
In `fmulero/labeler@1.0.5` the issue reported in https://github.com/andymckay/labeler/issues/40 is fixed. You can replace the `github-script` below with:
```yaml
        uses: fmulero/labeler@1.0.5
        with:
          repo-token: ${{ secrets.BITNAMI_BOT_TOKEN }}
          add-labels: verify
```